internal/locate/region_request.go

```diff
@@ -1735,6 +1741,10 @@ func (s *RegionRequestSender) onRegionError(
 		return false, nil
 	}
 
+	if isInvalidMaxTsUpdate(regionErr) {
```

Comment:
How about logging the request information here too?

Reply:
Makes sense